demo.js

```diff
@@ -5,12 +5,13 @@ const parse = require('regjsparser').parse;
 const generate = require('regjsgen').generate;
 const regenerate = require('regenerate');
 
-const pattern = String.raw`-`;
+const pattern = '\\w';
```

Comment:
I would’ve been fine with 

```
String.raw`\w`
```

as well but as long as the example is so small/simple, either works

Reply:
Oh actually I used `demo.js` to debug the code, I didn't realize I committed the changes I made.